ext/openssl/tests/openssl_random_pseudo_bytes_error.phpt

```
@@ -11,4 +11,4 @@ try {
 }
 ?>
 --EXPECT--
-openssl_random_pseudo_bytes(): Argument #1 ($length) must be greater than 0
+Length must be greater than 0
```

@kocsismate Is this change right?

---

No.  That looks like some sort of squash error on my part.  Thanks.  Will correct.

---

Actually I take it back.  That change is correct.  Somewhere along the way the output did change.  When I revert and make test I get:

---- EXPECTED OUTPUT
openssl_random_pseudo_bytes(): Argument #1 ($length) must be greater than 0
---- ACTUAL OUTPUT
Length must be greater than 0
---- FAILED

---

reverted.